modules/mono/editor/GodotTools/GodotTools/Build/BuildProblemsView.cs

```diff
@@ -0,0 +1,683 @@
+                HideRoot = true,
+            };
+            _problemsTree.ItemActivated += GoToSelectedProblem;
+            _problemsTree.ItemMouseSelected += ShowProblemContextMenu;
+            vbLeft.AddChild(_problemsTree);
+
+            // Problem context menu.
+            _problemsContextMenu = new PopupMenu();
+            _problemsContextMenu.IdPressed += ProblemContextOptionPressed;
+            _problemsTree.AddChild(_problemsContextMenu);
+
+            // Search box.
+            _searchBox = new LineEdit
+            {
+                SizeFlagsHorizontal = SizeFlags.ExpandFill,
+                PlaceholderText = "Filter Problems".TTR(),
+                ClearButtonEnabled = true,
+            };
+            _searchBox.TextChanged += SearchTextChanged;
+            vbLeft.AddChild(_searchBox);
+
+            var vbRight = new VBoxContainer();
+            AddChild(vbRight);
+
+            // Tools grid.
+            var hbTools = new HBoxContainer
+            {
+                SizeFlagsHorizontal = SizeFlags.ExpandFill,
+            };
+            vbRight.AddChild(hbTools);
+
+            // Clear.
+            _clearButton = new Button
+            {
+                Flat = true,
+                FocusMode = FocusModeEnum.None,
+                Shortcut = EditorDefShortcut("editor/clear_output", "Clear Output".TTR(), (Key)KeyModifierMask.MaskCmdOrCtrl | (Key)KeyModifierMask.MaskShift | Key.K),
+                ShortcutContext = this,
+            };
+            _clearButton.Pressed += Clear;
+            hbTools.AddChild(_clearButton);
+
+            // Copy.
+            _copyButton = new Button
+            {
+                Flat = true,
+                FocusMode = FocusModeEnum.None,
+                Shortcut = EditorDefShortcut("editor/copy_output", "Copy Selection".TTR(), (Key)KeyModifierMask.MaskCmdOrCtrl | Key.C),
+                ShortcutContext = this,
+            };
+            _copyButton.Pressed += CopySelectedProblems;
+            hbTools.AddChild(_copyButton);
+
+            // A second hbox to make a 2x2 grid of buttons.
+            var hbTools2 = new HBoxContainer
+            {
+                SizeFlagsHorizontal = SizeFlags.ShrinkCenter,
+            };
+            vbRight.AddChild(hbTools2);
+
+            // Toggle List/Tree.
+            _toggleLayoutButton = new Button
+            {
+                Flat = true,
+                FocusMode = FocusModeEnum.None,
+                TooltipText = GetToggleLayoutTooltipText(),
+                ToggleMode = true,
+                ButtonPressed = GetToggleLayoutPressedState(),
+            };
+            // Don't tint the icon even when in "pressed" state.
+            _toggleLayoutButton.AddThemeColorOverride("icon_pressed_color", Colors.White);
+            _toggleLayoutButton.Toggled += ToggleLayout;
+            hbTools2.AddChild(_toggleLayoutButton);
+
+            // Show Search.
+            _showSearchButton = new Button
```

Comment:
You need to replace flat with theme type variation `FlatButton`, so it has a nice pressed state.